src/Tizen.NET.MaterialComponents/Components/MNavigationDrawer.cs

```diff
@@ -9,7 +9,7 @@ public class MNavigationDrawer : MBox
         Box _mainContainer;
         Box _dimArea;
         EvasObject _main;
-        Panel _drawer;
+        protected Panel _drawer;
```

Comment:
Use property instead. And do not change the file mode.

Reply:
`Panel` will be protected property.